src/System.Private.CoreLib/src/System/Resources/ManifestBasedResourceGroveler.cs

```diff
@@ -421,7 +421,10 @@ private Assembly GetSatelliteAssembly(CultureInfo lookForCulture)
 
         // Perf optimization - Don't use Reflection for most cases with
         // our .resources files.  This makes our code run faster and we can avoid
-        // creating a ResourceReader via Reflection.  
```

Comment:
I will undo this in a later commit called "[Applied code review comments](https://github.com/dotnet/corert/pull/5904/commits/56fe44edc7df452bcf2b2f096c35292be2030de3)". Refer to this comment: https://github.com/dotnet/coreclr/pull/18283#discussion_r192810690